### ISSUE TITLE
Slashes in code block improperly rendered in preview

### ISSUE THREAD

**Issue by dumky** [prose/prose#874]
A forward slash within a code block (shifted by four spaces) gets previewed as "&#x2F;".
But a forward slash outside of a code block previews fine.
Also, the final rendering (by publishing the post) is correct.
Finally, for comparison, the preview in the GitHub editor for inputting this issue works properly.

test/test (this previews properly)

    test/test (this previews incorrectly)

A few examples within this post: https://github.com/dumky/dumky.github.io/blob/master/_posts/2015-08-14-git-internals.md

**Comment by phillipadsmith** [comment]
@dumky: Is this the preview issue that you're experiencing?

<img width="722" alt="previewing__posts_2015-08-08-your-filename_md_2015-08-08-your-filename_md_at_gh-pages_ _prose" src="https://cloud.githubusercontent.com/assets/34128/9290849/d8b7a698-4356-11e5-8188-a3b675414197.png">

**Comment by dumky** [comment]
Yes, exactly. 



—

**Comment by JohannesMP** [comment]
Similarly, less than/greater than characters are incorrectly encoded as the html entities `&lt;` and `&gt;` - so "`std::cout << "hello world" << std::endl;`" when typed in the editor becomes "`std::cout &lt;&lt; &quot;hello world&quot; &lt;&lt; std::endl;`".

However luckily this only occurs in the preview, and the actual text file is stored with the original characters

**Comment by dereklieu** [comment]
@JohannesMP Yeah, wonder if it has something to do with a very simple escaping scheme used to generate preview content. Probably related to #837

**Comment by haschdl** [comment]
Just tried out the tool today, and I noticed this in the first preview page I opened. So it seems this is still an issue? @phillipadsmith
